### DISCUSSION TITLE
Tutorial?

### DISCUSSION THREAD

**Discussion by mandomal** [fenix-hub/godot-engine.easy-charts#47] [Q&A]
Very cool project. Is there some sort of video or written guide on how to go about using this package? 
I've gotten the example data to work with the linechart but I'm not really sure where to go from here. 

Here are some things I would like to do but don't know where to start:
1. Plot a monocolor histogram
2. Add x and y labels
3. Specify the line color.
4. Change the line width and marker of the points.
5. Add antialiasing to the lines.
6. Animate data being added.

Also I had to make the utilities script an autoload to get the code below to work:
$LineChart.plot_from_csv("path/to/example/data.csv",";")

Is this the normal way to use the package? If so, do I need to assign more autoloads for my project?

**Comment by fenix-hub** [comment]
@mandomal 
Hope you are doing well and still interested in this plugin.
There are now some examples in the plugin folder, under `examples` which cover most of your requests.
Since some features are going to be added step by step, let me know if you are able to achieve what you want, and if not, how can the plugin be improved.
